chaos/cluster.go

```diff
@@ -0,0 +1,258 @@
+package chaos
```

Comment:
@xiang90 @diegs 
* I think the motive is to allow rebooting the entire cluster / nodes. Do you recommend changing the name?